src/test/java/org/apache/commons/io/IOUtilsTest.java

```diff
@@ -1716,4 +1716,18 @@ public void testWriteLittleString() throws IOException {
         }
     }
 
+    @Test
+    public void testByteArrayWithIllegalSize() {
+        try {
+            int size = -1;
+            byte[] bytes = IOUtils.byteArray(size);
+
+            size = 0;
+            bytes = IOUtils.byteArray(size);
+        }catch (Exception e) {
+            assertEquals(e.getClass().getName(), IllegalArgumentException.class.getName());
+        }
```

Comment:
I think instead it could use JUnit's `@Test(expected=IllegalArgumentException.class)`, or `assertThrows`.

Reply:
And I think when the first exception happens, the second is part of the test is never executed? Probably needs to be broken down into two separate tests, maybe with `@ParameterizedTest` too.

Reply:
ok, I changed IOUtilsTest.testByteArrayWithIllegalSize() with @ParameterizedTest.
you can review the code.

Reply:
Why would we trade a more vague exception for the more precise and quite specific one already thrown by the VM? Maybe this issue can be simplified with better Javadoc?

Reply:
Hi garydgregory,
    Thank you for your good suggestion, I updated the doc and changed the unittest, you can review it again.

Reply:
In order to improve my work efficiency，I will study github workflow in the next week.